File: C#/Udemy_Encapsulamento/ContaBancaria/ContaBancaria/Conta.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ContaBancaria
{
    public class Conta
    {
        public int Numero { get; set; }
        public string Titular { get; set; }
        public decimal Saldo { get; set; }

        public Conta() { 
        }

        public Conta(int numero, string titular)
        {
            this.Numero = numero;
            this.Titular = titular;            
        }        

        public void Depositar(decimal valor) {
            Saldo += valor;
        }

        public void Sacar(decimal valor)
        {
            Saldo -= valor;
        }

        public override string ToString() {
            return $"Conta {Numero}, Titular: {Titular}, Saldo: $ {Saldo}";            
        }
    }
}

File: Java/04_Udemy_Heranca/src/entidades/Conta.java
package entidades;

public class Conta {
	private Integer numero;
	private String titular;
	protected double saldo;
	
	public Conta() {
		
	}

	public Conta(Integer numero, String titular, double saldo) {
		super();
		this.numero = numero;
		this.titular = titular;
		this.saldo = saldo;
	}

	public Integer getNumero() {
		return numero;
	}

	public void setNumero(Integer numero) {
		this.numero = numero;
	}

	public String getTitular() {
		return titular;
	}

	public void setTitular(String titular) {
		this.titular = titular;
	}

	public double getSaldo() {
		return saldo;
	}
	
	//saca e desconta 5 de taxa
	public void sacar(double valor) {
		saldo -= valor + 5.0;
	}
	
	public void depositar(double valor) {
		saldo += valor;
	}
	
	
	
}
